packages/config-manager/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Hash, HexNumber } from "@ckb-lumos/base";
+
+export interface Config {
+  rollupTypeHash: Hash;
+  polyjuice: PolyjuiceConfig;
+}
+
+export interface PolyjuiceConfig {
+  // type hash
+  scriptCodeHash: Hash;
```

Comment:
is scriptCodeHash the creator script type hash?

```suggestion
  creatorScriptHash: Hash;
```

Reply:
No, it's polyjuice validator script type hash.

Reply:
I see,  from [web3](https://github.com/nervosnetwork/godwoken-web3/blob/main/packages/api-server/src/methods/modules/poly.ts#L161)
) , should we rename this variable to 

```suggestion
  validatorTypeHash: Hash;
```